packages/codec/lib/storage/allocate/index.ts

```diff
@@ -69,46 +80,62 @@ export function getStorageAllocations(
       }
     }
   }
-  for (const contract of Object.values(contracts)) {
+  return allocations;
+}
+
+export function getStateAllocations(
```

Comment:
this might be worth a doc comment saying what "state" means

Reply:
OK, I can add that.